CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+Table of Contents
+=================
+* [Code of Conduct of Eclipse Che documentation project](#code-of-conduct-of-eclipse-che-documentation-project)
+  * [Our Pledge](#our-pledge)
+  * [Our Standards](#our-standards)
+  * [Attribution](#attribution)
+* [Pull Request Process](#pull-request-process)
```

Comment:
I would suggest that the ToC follows a typical workflow, so talk about Creating documentation first, then building, then submitting a PR, then publishing.